### PR TITLE
Update Plugin.php

### PR DESCRIPTION
**Pull request by jomonkj**
#6 fIX

### CODE REVIEW

**Comment by hiqsol** [comment]
Ok, thank you.
Good idea, I'll take it soon.

File: src/Plugin.php [hunk 1]
```diff
@@ -180,6 +180,13 @@ protected function processPackage(CompletePackageInterface $package)
             'aliases' => $aliases,
         ]);
     }
+    
+    protected function getExtra(CompletePackageInterface $package)
+    {
+        $path = $this->preparePath($package,'composer.json');
+        
+        return (json_decode(@file_get_contents($path), true))['extra'] ?? [];
```
Comment (SilverFire) [inline src/Plugin.php:188]: @hiqsol I think it makes sense to load that composer.json into a Package object

**Review by SilverFire — CHANGES_REQUESTED** [review]

**Comment by hiqsol** [comment]
I think to extract all package related logic to `Package` class wrapper around composer Package class.

Apropos, development moved to 1.0 branch - master is freezed.

@jomonkj do you need this change now? I'll do it. But later. Ok?

**Comment by jomonkj** [comment]
Ok, No problem
I can manage with forked repo

**Comment by hiqsol** [comment]
Ok :)
I've managed to merge 1.0 to master and it seems to work.
I'll implement reading extra from file soon (today).

**Comment by hiqsol** [comment]
I've rewritten package logic to prefer data from `composer.json` where suitable.
Should be fine now. So closing this.

**Pull request closed without merging**